File: planet-render/src/au/com/codeka/planetrender/TemplateException.java
package au.com.codeka.planetrender;

public class TemplateException extends Exception {
    private static final long serialVersionUID = 1L;

    public TemplateException() {
    }

    public TemplateException(Throwable cause) {
        super(cause);
    }

    public TemplateException(String message) {
        super(message);
    }

    public TemplateException(String message, Throwable cause) {
        super(message, cause);
    }
}

File: planet-render/src/main/java/au/com/codeka/warworlds/planetrender/TemplateException.kt
package au.com.codeka.warworlds.planetrender

class TemplateException : Exception {
  constructor() {}
  constructor(cause: Throwable?) : super(cause) {}
  constructor(message: String?) : super(message) {}
  constructor(message: String?, cause: Throwable?) : super(message, cause) {}

  companion object {
    private const val serialVersionUID = 1L
  }
}